### DISCUSSION TITLE
win10安装了IDM6.39.7，谷歌浏览器下鼠标手势没有效果

### DISCUSSION THREAD

**Discussion by tangtanag** [shuax/MouseInc.Settings#184] [Q&A]
我原本用的是IDM6.38.1版本，MouseInc可以正常用，换成IDM6.39.7之后鼠标手势失效了（浏览器界面长按鼠标右键拖动，没出现棕色笔画），卸载IDM6.39.7之后MouseInc又可以用了。有相同情况的小伙伴吗

**Comment by Hi-John** [comment]
我edge不能用手势了，也装了idm，估计和你这个情况一样，看来我也要换个idm版本了，还一直在排查是哪个插件的问题，终于找到了

**Comment by shuax** [comment]
试试管理员运行
